### ISSUE TITLE
Uncommon Normalization Error

### ISSUE THREAD

**Issue by lahart21** [netstim/leaddbs#748]
I am unable to run normalization on most of my datasets. Matlab shows a false error of ANT's failing due to memory problems, I changed the settings from 4 threads to 1 and this did not solve the problem. When I looked through the normalize log file I found that the actual error was in line 316 and Coregistration is failing and not initializing itself correctly. Need to switch the MRI to MRI method from SPM to ANT's and run Coregistration separately from normalization and brain shift correction. 

Exception caught: 
itk::ExceptionObject (0x7fb997a17bc0)
Location: "unknown" 
File: /Users/ningfei/Desktop/ANTs/build/staging/include/ITK-5.2/itkMattesMutualInformationImageToImageMetricv4.hxx
Line: 316
Description: itk::ERROR: itk::ERROR: MattesMutualInformationImageToImageMetricv4(0x7fb99670ac10): All samples map outside moving image buffer. The images do not sufficiently overlap. They need to be initialized to have more overlap before this metric will work. For instance, you can align the image centers by translation.

**Comment by simonoxen** [comment]
Hi Lauren, as discussed, the source of issue here was in the coregistration, which is solved (as you mention) by changing the coregestration strategy. One could check coreg before running normalization so as to catch the coreg misalignment early on.

I'd be surprised if this really happens in most of your datasets. If so, perhaps switch to ANTs as default for MR->MR which seemed to worked better in that case.

**Comment by lahart21** [comment]
Yes thank you, and you are correct I should have gone into the normalize log file to verify that this was the error. Is there anyway we could make it so that lead doesn't throw the incorrect error of a memory problem and instead shows that the error has to do with initializing co-registration? 

It's my understanding that normalization fails due to a 3rd party error, and it's not easy for lead- to catch the exception, since the code being executed is through the 3rd party code. 

Maybe we could change error warnings to give a list of options users can type?

**Comment by jAchtzehn** [comment]
At least the error should not default to saying it is a memory problem, which I think it currently does!

**Comment by ningfei** [comment]
agreed that maybe better throw out more relevant error information. Thought it's already done, but turns out it's still in my list...
